processing/data_collection/gazette/spiders/ce_fortaleza.py

```diff
@@ -0,0 +1,55 @@
+import re
+import dateparser
+import w3lib.url
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+
+
+class CeFortalezaSpider(Spider):
+    GAZETTE_ELEMENT_CSS = '.diarios-oficiais .table-responsive tbody tr'
+    DATE_CSS = 'td:nth-child(2)::text'
+    EXTRA_CSS = 'td:nth-child(1)::text'
+    NEXT_PAGE_CSS = 'ul.pagination .page-link::attr(href)'
+
+    MUNICIPALITY_ID = '2304400'
+
+    allowed_domains = ['apps.fortaleza.ce.gov.br']
+    name = 'ce_fortaleza'
+    start_urls = ['http://apps.fortaleza.ce.gov.br/diariooficial/?mes-diario=todos']
+
+    def start_requests(self):
+        requests = []
+
+        for year in range(2015, datetime.now().year + 1):
+            year_url = w3lib.url.add_or_replace_parameter( self.start_urls[0], 'ano-diario', year)
```

Comment:
`start_urls` is just a shortcut to `start_requests`. As you are using it, I think i'd be better to remove `start_urls` and create a variable `base_url` and use it:

```
def start_requests(self):
    base_url = 'http://apps.fortaleza.ce.gov.br/diariooficial/?mes-diario=todos'
    (...)
    for ...
        year_url = w3lib.url.add_or_replace_parameter(base_url, 'ano-diario', year)
```

Reply:
Remove extra space:
`year_url = w3lib.url.add_or_replace_parameter(self.start_urls[0], 'ano-diario', year)`